test/github-issues/7605/issue-7605.ts

```diff
@@ -0,0 +1,38 @@
+import "reflect-metadata";
+import {
+    closeTestingConnections,
+    createTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils";
+import { Connection } from "../../../src/connection/Connection";
+import { Foo } from "./entity/foo";
+import { expect } from "chai";
+
+describe("github issues > #7605 Giving 'column \"response.created\" must appear in the GROUP BY clause or be used in an aggregate function', but it does", () => {
+    let connections: Connection[];
+    before(
+        async () =>
+            (connections = await createTestingConnections({
+                entities: [__dirname + "/entity/*{.js,.ts}"],
+                enabledDrivers: ["postgres", "cockroachdb", "mssql"],
+            }))
+    );
+    beforeEach(() => reloadTestingDatabases(connections));
+    after(() => closeTestingConnections(connections));
+
+    it("should use the same parameter index for all ocurrences of a parameter on drivers that use indexed parameters", () =>
```

Comment:
Please add a test case for other scenarios that could be impacted - either in this test file or elsewhere.

In particular:

* Array placeholders
* Using some placeholders multiple times and others not
* Using multiple placeholders multiple times